client/src/main/java/org/schlunzis/kurtama/client/util/I18nUtils.java

```diff
@@ -0,0 +1,136 @@
+package org.schlunzis.kurtama.client.util;
+
+import javafx.beans.binding.Bindings;
+import javafx.beans.binding.StringBinding;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.scene.control.Tooltip;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.text.MessageFormat;
+import java.util.*;
+
+
+/**
+ * Utility-Class to support internationalization.
+ *
+ * <br>
+ * Some research pointed out the eclipse solution for i18n. For further
+ * information have a look in the <a href=
+ * "https://www.eclipse.org/eclipse/platform-core/documents/3.1/message_bundles.html">documentation</a>.
+ * <br>
+ * Nevertheless, it is not used, as it is a completely different approach then
+ * previously used and does not bring that much of an advantage. <br>
+ * Further best practices have been discussed in <a href=
+ * "https://stackoverflow.com/questions/6992977/where-to-place-i18n-key-strings-in-java">this</a>
+ * Stack Overflow article.
+ *
+ * @author Jonas Pohl
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class I18nUtils {
+
+    /**
+     * The resource bundle on which elements can bind in order to be updated on
+     * language change.
+     */
+    private static final ObjectProperty<ResourceBundle> bundle = new SimpleObjectProperty<>();
+
+    /*
+     * The supported locales.
+     * TODO: determine the supported locales from the language files at runtime.
+     */
+    @Getter
+    private static final List<Locale> SUPPORTED_LOCALES = Arrays.asList(Locale.GERMANY, Locale.US);
+
+    /**
+     * The current locale. Will be initially set to systems default ({@link Locale#getDefault()}).
+     */
+    @Getter
+    private static Locale locale;
+
+    static {
+        // setLocale(SUPPORTED_LOCALES.getFirst());
+        setLocale(Locale.getDefault());
```

Comment:
You could use the user settings to save the selection for the next time.

Reply:
Done!